crawler.rb

```diff
@@ -0,0 +1,11 @@
+require 'fileutils'
```

Comment:
We need to add `require 'relaton-iana'` to be able to call the `RelatonIana::DataFetcher.fetch` directly.